### ISSUE TITLE
[bitnami/joomla] Error: template: joomla/templates/NOTES.txt:73:28: executing "joomla/templates/NOTES.txt" at <include "common.validations.values.mariadb.passwords" 

### ISSUE THREAD

**Issue by ceejoyce** [bitnami/charts#3589]
**Which chart**:
[bitnami/joomla] version 8.1.2

**Describe the bug**
Can't deploy chart, seems to be related to #3557 when using `mariadb.enabled=false` and setting externalDatabase values.

```Error: template: joomla/templates/NOTES.txt:73:28: executing "joomla/templates/NOTES.txt" at <include 
"common.validations.values.mariadb.passwords" (dict "secretName" (include "joomla.mariadb.fullname" .) "context" $)>: error calling include: template: joomla/charts/common/templates/_validations.tpl:68:98: executing "common.validations.values.mariadb.passwords" at <.context.Values.mariadb.secret.requirePasswords>: nil pointer evaluating interface {}.requirePasswords
```

**To Reproduce**
`helm install joomla-test --set joomlaUsername=admin,joomlaPassword=password,mariadb.enabled=false`

**Expected behavior**
The chart to deploy.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.2.3", GitCommit:"8f832046e258e2cb800894579b1b3b50c2d83492", GitTreeState:"clean", GoVersion:"go1.13.12"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:26:26Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.7", GitCommit:"169db3bff4b5fb7722e967c5b6356713f05f15ed", GitTreeState:"clean", BuildDate:"2020-04-03T16:14:09Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

**Comment by joancafom** [comment]
Hi @ceejoyce 

Thanks for your issue, I have been able to reproduce it. It seems that the PR solving this has already been merged #3565 . The new Joomla Chart version 8.1.3 that is going to be released soon should ship with those changes that will hopefully solve this problem! 

Thanks!

**Comment by joancafom** [comment]
Hi @ceejoyce 

Just to let you know that version `8.1.3` was released! 

```console
$ helm repo update && helm search repo bitnami/joomla
NAME          	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/joomla	8.1.3        	3.9.21     	PHP content management system (CMS) for publish...
```

This should solve your issue. Thanks!
